### ISSUE TITLE
Record Stroke Pressure

### ISSUE THREAD

**Issue by albertotono** [google/monster-mash#14]
It would be nice to record the stroke pressure of the designer, currently every line has the same thickness. 

`    const int t = ceil(0.5 * thickness);`

https://github.com/google/monster-mash/blob/6467b2e4f183738128fea4d713c44f5de863f76e/src/mypainter.cpp#L109

What would be the best way to approach this task? C++ or .js?

**Comment by DominikNovosel** [comment]
Is there any particular reason why their thickness would be important? The lines only define the contour of the blobs on the model and are not visible on the final product.

**Comment by albertotono** [comment]
Definitely, they are extremely important. See Brunelleschi and Da Vinci for references.

**Comment by DominikNovosel** [comment]
I think you misunderstood. I said I don't understand how this app would benefit from recognizing stroke pressure.
